extra-scientific/boinc/autobuild/overrides/usr/share/applications/boinc.desktop

```diff
@@ -1,19 +1,22 @@
 [Desktop Entry]
 Type=Application
 Version=1.0
-Exec=/usr/bin/boincmgr
+Exec=/usr/bin/boincmgr -d /var/lib/boinc
 Path=/var/lib/boinc
```

Comment:
I just rechecked it again and it actually worked... Sorry for not notifying that.